IdokladSdk/Models/ReceivedInvoice/List/ReceivedInvoiceItemListGetModel.cs

```diff
@@ -60,6 +60,11 @@ public class ReceivedInvoiceItemListGetModel : IEntityId
         [StringLength(20)]
         public string Unit { get; set; }
 
+        /// <summary>
+        /// Gets or sets vat code id.
+        /// </summary>
+        public int? VatCodeId { get; set; }
```

Comment:
VatCode do Expandu nepříjde?

Reply:
Správná otázka, ale aktuálně to nemáme v žádných entitách. Mělo by se to pak udělat hromadně.

Reply:
ok